.pre-commit-config.yaml

```diff
@@ -27,6 +27,10 @@ repos:
           # Configure Black to support only syntax supported by the minimum supported Python version in setup.py.
           - --target-version=py37
         files: \.py$|\.pyi$
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.2
+    hooks:
+    -   id: pyupgrade
```

Comment:
```suggestion
      - id: pyupgrade
        args:
          - --py37-plus
```
Thanks to this, we will have a few more improvements, in particular [f-strings](https://github.com/asottile/pyupgrade#f-strings).